drivers/hmis/app/models/hmis/form/assessment_processor.rb

```diff
@@ -55,49 +68,43 @@ def enrollment_coc_factory(create: true)
 
     self.enrollment_coc = enrollment_factory.enrollment_cocs.
       build(
-        data_collection_stage: assessment_detail.data_collection_stage,
         household_id: enrollment_factory.household_id,
         project_id: enrollment_factory.project_id,
-        personal_id: enrollment_factory.client.personal_id,
-        information_date: assessment_detail.assessment.assessment_date,
-        user_id: assessment_detail.assessment.user_id,
+        **common_attributes,
       )
   end
 
+  def exit_factory(create: true)
+    return exit if exit.present? || !create
```

Comment:
this is _probably_ ok, but `exit` is a reserved work in ruby, so you _may_ need to: 
```suggestion
    return self.exit if self.exit.present? || !create
```

Reply:
Seems to work fine, but I can make that change